### PR TITLE
Move ingress to workers

### LINKED ISSUE #75: Move work loads and ingress to workers
Right now we use the main master as both the contact point for kubectl and also the entry point for ingress. This is simpler for dev work but in reality we don't want the control plane also routing ingress workloads. We should limit this to the worker nodes. And also we maybe should respect taints and only assign workloads to workers. This is how it should be in production.

This will touch the ingress ansible role, some other roles, the terraform, and maybe a few other places.

### PR DESCRIPTION
**Pull request by aperullo**
Closes #75 

This change keeps ingress pods from being scheduled on master nodes, that way only workers route traffic, a security improvement.

This change does _not_ prevent masters from hitting and resolving dns/ingress names internally, as that would break their ability to pull from the cluster's internal registry.

### CODE REVIEW

**Comment by njbarber** [comment]
I looked at the issue this closes. Do you have additional information on ` we don't want the control plane also routing ingress workloads`?  Just trying to understand how much of an issue this really is. If it's significant, I'm surprised the `nginx-ingress` chart doesn't apply or provide options for applying annotations itself.

**Comment by njbarber** [comment]
Never mind. I hadn't reviewed the changes closely. I thought you were adding annotations, not removing them...

**Comment by pvargas** [comment]
Why did you replace `_` with `-`?

**Comment by aperullo** [comment]
To make the naming consistent on aws

File: kubernetes/terraform/network/security_groups.tf [hunk 1]
```diff
@@ -11,14 +11,14 @@ resource "aws_security_group" "ports" {
   }
 
   tags = {
-    Name = "${terraform.workspace}_${var.name}_ports"
```
Comment (njbarber) [inline kubernetes/terraform/network/security_groups.tf]: Looks like you missed a spot at the very top of this file: `name` should be `${terraform.workspace}-${var.name}-ports`.

Comment (aperullo) [reply kubernetes/terraform/network/security_groups.tf]: Yep, missed a spot. Got it now, thanks!

**Pull request merged**